Dockerfile.mysql

```diff
@@ -0,0 +1,8 @@
+ARG DOCKER_TAG=9.1.0
```

Comment:
The specified MySQL version '9.1.0' does not exist. Consider using a valid version such as '8.0' or '5.7'.